src/reducers/gui.js

```diff
@@ -104,6 +106,7 @@ const guiReducer = combineReducers({
     hoveredTarget: hoveredTargetReducer,
     stageSize: stageSizeReducer,
     menus: menuReducer,
+    micIndicator: micIndicatorReducer,
```

Comment:
Might want to name this micIndicatorOn to make it more clear what it means (since it's a boolean)